packages/fxa-content-server/app/tests/spec/views/post_verify/third_party_auth/set_password.js

```diff
@@ -0,0 +1,147 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import _ from 'underscore';
+import { assert } from 'chai';
+import Account from 'models/account';
+import Backbone from 'backbone';
+import BaseBroker from 'models/auth_brokers/base';
+import Metrics from 'lib/metrics';
+import Relier from 'models/reliers/relier';
+import SentryMetrics from 'lib/sentry';
+import sinon from 'sinon';
+import User from 'models/user';
+import View from 'views/post_verify/third_party_auth/set_password';
+import WindowMock from '../../../../mocks/window';
+import $ from 'jquery';
+
+const PASSWORD_INPUT_SELECTOR = '#password';
+const VPASSWORD_INPUT_SELECTOR = '#vpassword';
```

Comment:
I left a note above about import/exporting these from https://github.com/mozilla/fxa/blob/3d66e9e888e541e413e50c9c02a8f232e507d1ea/packages/fxa-content-server/app/scripts/views/post_verify/third_party_auth/set_password.js#L17-L18

But maybe they're far enough apart (maybe `../../../../scripts/views/post_verify/third_party_auth/set_password.js`) that hardcoding them in each file is better. 🤷